pkgs/by-name/ni/nimlsp/package.nix

```diff
@@ -1,10 +1,10 @@
-{ lib, buildNimPackage, fetchFromGitHub, srcOnly, nim-unwrapped-1 }:
+{ lib, buildNimPackage, fetchFromGitHub, srcOnly, nim-unwrapped-2 }:
 
 buildNimPackage (finalAttrs: {
   pname = "nimlsp";
   version = "0.4.6";
 
-  requiredNimVersion = 1;
+  requiredNimVersion = 2;
```

Comment:
You can remove this entirely since the default is 2: https://github.com/NixOS/nixpkgs/blob/5b321ac6b3acbfa5c8458ee3701db18d530f3a17/pkgs/development/compilers/nim/build-nim-package.nix#L8